doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,54 @@
+.. role:: html(raw)
+   :format: html
+
+.. _embeddings:
+
+Quantum embeddings
+===================
+
```

Comment:
I suggest moving the last sentence of the following paragraph to the top, so it is more visible. I.e.,

```suggestion

.. seealso:: PennyLane provides built-in embedding templates; see :mod:`pennylane.templates.embeddings` for more details.

```

Reply:
I put it at the end of the page so it does not distract away from the concept section before it has to :) check the new commit.  

I also added a `see also` section in concepts/quantum nodes at the end of its corresponding page.

Reply:
:+1: